### PR TITLE
feat: avatar and badge DONE

### PR DESCRIPTION
**Pull request by maitrakhatri**
Please review the code and do let me know if it needs any changes 

image

image

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

File: badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    text-align: center;
+    border-radius: 50%;
+}
+
+.red {
+    background-color: var(--danger);
+}
+
+.blue {
+    background-color: var(--secondary);
```
Comment (anupkgurung) [inline badge/badge.css:26]: good to use color name for color instead of property.

File: avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@import url(../style.css);
+
+.avatar {
+    height: 32px;
+    width: 32px;
+}
+
+.avatar-xl {
+    height: 48px;
```
Comment (anupkgurung) [inline avatar/avatar.css:9]: avoid using px as it might trouble you when doing responsiveness.

**Review by anupkgurung — COMMENTED** [review]
rest seems good.

File: badge/badge.css (hunk 1 above)
Comment (maitrakhatri) [reply badge/badge.css:26]: yes, I need to change the colors name in root CSS file

File: avatar/avatar.css (hunk 2 above)
Comment (maitrakhatri) [reply avatar/avatar.css:9]: okay, I'll take care of it. Thank you  !!

**Pull request merged**